SEP/SEP-0003/sep-0003.md

```diff
@@ -0,0 +1,44 @@
+# Property paths with a min/max hop
+
+* Short Name: min-max-propery-path
+* SEP Number: [SEP-0003](sep-0003.md)
+* Author: [Jerven Bolleman](https://github.com/jervenbolleman)
+
+## Abstract
+
+In the SPARQL 1.1. working group, a form of bounded property path where supported in the late [drafts](https://www.w3.org/TR/2012/WD-sparql11-query-20120105/#propertypaths). The form being a path element _elt_, followed by a curly bracket enclosed min/max pattern.
+
+||Syntax Form||Matches||
+| _elt_{n,m} | A path of between n and m occurrences of elt. |
+| _elt_{n}   | A path of exactly n occurrences of elt. |
+| _elt_{n,}  | A path of n or more occurrences of elt. |
+| _elt_{,n}  | A path of between 0 and n occurrences of elt. |
+
+This was removed in later draft and recommondation, upon the change to non-counting semantics.
+
+> Such connectivity matching does not introduce duplicates (it does not incorporate any count of the number of ways the connection can be made) even if the repeated path itself would otherwise result in duplicates. 
+
+The functionality of limited connectivity checks is still valid with non-counting semantics, in pragmatic terms. 
+
+## Motivation
+
+As raised in [sparql-12#101](https://github.com/w3c/sparql-12/issues/101), it would be nice to have this functionaliy available in SPARQL 1.2.
```

Comment:
functionality
```suggestion
As raised in [sparql-12#101](https://github.com/w3c/sparql-12/issues/101), it would be nice to have this functionality available in SPARQL 1.2.
```